Tests/BugsnagConfigurationTests.m

```diff
@@ -1,5 +1,6 @@
 #import "Bugsnag.h"
 #import "BugsnagConfiguration.h"
+#import "BugsnagKeys.h"
```

Comment:
We should not use the keys file within the tests. The tests should validate values rather than show equality with the values being tested. For example, if a typo occurred in the keys file changing the value of `BSGKeyType` to `"typ"`, the test would not break but should.